docs/src/components/home/DiamondSponsors.tsx

```diff
@@ -15,14 +15,14 @@ const DIAMONDs = [
     srcSet: 'https://avatars3.githubusercontent.com/u/1287123?s=80 2x',
     name: 'Octopus Deploy',
     description: 'Repetable relayable deployments.',
-    href: 'https://octopus.com/',
+    href: 'https://octopus.com/?utm_medium=referral&utm_source=mui.com&utm_content=homepage',
```

Comment:
We have used this in the other instance of the codebase

```suggestion
    href: 'https://octopus.com/?utm_medium=referral&utm_source=materialui&utm_content=homepage',
```

how about we replace everything to
```suggestion
    href: 'https://octopus.com/?utm_medium=referral&utm_source=mui&utm_content=homepage',
```

Reply:
Also, using utm_content over utm_campaign does sound better. Could we update the other links? (README.md and https://next.material-ui.com/discover-more/backers/

Reply:
sure.